docs/tutorials/structure_function_showcase.ipynb

```diff
@@ -10,33 +10,85 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # amp is RMS amplitude of the process and tau is the decorrelation timescale
</pre><p>It is not clear what "RMS amplitude" means, sounds like a mix of "RMS of the process" and "characteristic amplitude of the light curve".</p><p> </p><p>Also, to make it more astro, lets specify units maybe? Like amplitude in magnitudes and timescale in days?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I though about that. I prefer to leave it unitless to avoid some potential confusion if the lightcurve generated have units (they do not).